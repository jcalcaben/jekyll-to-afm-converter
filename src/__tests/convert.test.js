const convert = require("../convert");

test("leaves normal markdown alone", () => {
  const content = "**bold** _italics_";

  const result = convert({
    fileContent: content,
  });

  expect(result).toMatchSnapshot();
});

test("fix open HTML tags", () => {
  const content = `hello
  
<br>  

world`;

  const result = convert({
    fileContent: content,
  });

  expect(result).toMatchSnapshot();
});
