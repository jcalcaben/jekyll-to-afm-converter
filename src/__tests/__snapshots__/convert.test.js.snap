// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`fix open HTML tags 1`] = `
"hello

<br>  

world
"
`;

exports[`leaves normal markdown alone 1`] = `
"**bold** _italics_
"
`;
